lib/node_modules/@stdlib/string/next-grapheme-cluster-break/lib/main.js

```diff
@@ -65,13 +69,20 @@ function nextBreak( str, idx ) {
 	var mid;
 	var i;
 
+	if ( !isString( str ) ) {
+		throw new TypeError( 'invalid argument. Must provide a string. Value: `' + str + '`.' );
+	}
 	len = str.length;
-	if ( idx < 0 ) {
+	if ( len === 0 ) {
```

Comment:
Why return `0` instead of `-1`?

Reply:
If the string is empty, then it has a vacuous break at `str.length`, which in this case is `0`.
But thanks for pointing this out, an empty string should return `-1` for `prevGraphemeClusterBreak` as the vacuous break there is at `-1`. I had missed that.